test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1

```diff
@@ -114,5 +114,9 @@ Describe "Stream writer tests" -Tags "CI" {
             $result.Count | Should -Be $returnCount
             (Compare-Object $result $returnValue -SyncWindow 0).length | Should -Be 0
         }
+
+        It "Write-Information accespts `$Null" {
+            $null | Write-Information | Should -BeNullOrEmpty
```

Comment:
Rather than "NullOrEmpty", we should probably be specific about what this case does, and test it by checking the information stream for the result if possible.

Reply:
Not sure what you're proposing to check here.  Previously it was returning a non-terminating error.  I can change it to stop on error which would cause the test to fail without this change.  The `OrEmpty` of the assertion only applies if the input is a string anyways.

Reply:
`Write-Information` doesn't write to pipeline so the test is misleading. It is more clear to check Information stream contains $null or check `-NotThrow`.

Reply:
Ok, I see what you guys are saying now.  Will make a change.